### PR TITLE
add logistic regression

### PR DESCRIPTION
**Pull request by boyinggong**
I added the logistic regression using package sklearn to calculate the lambdas for each subject. There's also scores and cross validation scores for for each regression model.

### CODE REVIEW

**Comment by BenjaminHsieh** [comment]
Can we try to make this into a function, that takes parameters X, Y and outputs information we need, so lines 21-32

**Comment by BenjaminHsieh** [comment]
how is this going? do we want to leave this as a script or write functions to wrap somethings together?

**Comment by boyinggong** [comment]
Add linear mixed effects functions and scripts. please check it :)

File: code/utils/lme_functions.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
+from __future__ import division
+from statsmodels.regression.mixed_linear_model import MixedLM
+import pandas as pd
+import matplotlib.pyplot as plt
+import nibabel as nib
+import numpy as np
+from behavtask_tr import events2neural_extend, merge_cond
+from regression_functions import hrf, getGainLoss, calcBeta, calcMRSS, deleteOutliers
```
Comment (BenjaminHsieh) [inline code/utils/lme_functions.py]: No more getGainLoss, change to getRegressor

File: code/utils/lme_script.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
+from __future__ import division
+from statsmodels.regression.mixed_linear_model import MixedLM
+import pandas as pd
+import matplotlib.pyplot as plt
+import nibabel as nib
+import numpy as np
+from behavtask_tr import events2neural_extend, merge_cond
+from regression_functions import hrf, getGainLoss, calcBeta, calcMRSS, deleteOutliers
```
Comment (BenjaminHsieh) [inline code/utils/lme_script.py]: same thing here

**Comment by BenjaminHsieh** [comment]
In logistic.py, we need a string that specifies where the data is, ie a pathtodata.
Same thing in lme_script.py, it is better to specify where the data files are, not change the directory into the data folder (line 21). Also can you move this script under the scripts folder in code? thanks

**Comment by BenjaminHsieh** [comment]
move logistic.py to script?

**Comment by BenjaminHsieh** [comment]
add need create a pathtodata, something like:
\textbf{pathtodata = '../../data'}

add pathtodata whenever you load in a txt file

**Comment by BenjaminHsieh** [comment]
- pathtodata = '../../data'

**Comment by boyinggong** [comment]
fixed path problem.

**Comment by briankleinqiu** [comment]
Since it seems like it's going to write a bunch of files can you make a folder in data directory to dump it into?

you can use 
if not os.path.exists(directory):
    os.makedirs(directory)
to check if it exists and then make it, but it's probably easier just to do it manually lol

**Comment by BenjaminHsieh** [comment]
merging for now, txt files saved under data, can update later to save in a separate directory

**Pull request merged**